docs/src/Installation.md

```diff
@@ -0,0 +1,45 @@
+# Install Julia
+RMS is written in Juila language. So before stepping any further, Julia needs to be installed. The download links can be found at [download Julia](https://julialang.org/downloads/). More instructions can be found from the [instruction page](https://julialang.org/downloads/platform/).
+
+## Standard Installation
+With julia RMS can be installed with:  
+
+```
+using Pkg
+Pkg.add("ReactionMechanismSimulator")
+Pkg.build("ReactionMechanismSimulator")
+```
+
+## Developer Installation
+Clone RMS to your machine in an appropriate location we will refer to as `RMS_PATH``:  
+```
+git clone https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl.git
+```
+then you can install with 
+```
+import Pkg
+Pkg.develop(Pkg.PackageSpec(path=RMS_PATH)) 
+Pkg.build("ReactionMechanismSimulator")
+```
+
+## Testing RMS
+Unit and functional tests for RMS can be run with: 
+```
+import Pkg
+Pkg.test("ReactionMechanismSimulator")
+```
+
+## pyrms
+We also provide a python wrapper for RMS, [pyrms](https://github.com/ReactionMechanismGenerator/pyrms). Installation instructions are available on its github page. 
+
+## Julia-Python Linking
```

Comment:
Maybe move this above the developer installation? Or does `Pkg.build("ReactionMechanismSimulator")` handle the environment installation?

Reply:
Linking to the default python env julia creates has always been automatic. We did use this procedure before for installation when developing RMG and RMS simultaneously, but I don't think we even do this explicitly anymore because I believe julia.install() automatically links the correct python and julia without need for this. However, the directions do tend to be useful to those of us that are messing around with RMG and RMS simulatneously.